securitas/templates/password-reset.html

```diff
@@ -1,11 +1,6 @@
-{% extends "with_navbar.html" %}
+{% extends "main.html" %}
```

Comment:
This form still need adapting to Bootstrap

Reply:
resolved by:
https://github.com/fedora-infra/securitas/pull/92/commits/3430e4eae94c526a82068d42e9710023727a2ba7